.all-contributorsrc

```diff
@@ -29,5 +38,6 @@
   "projectName": "eslint-plugin-never-disable",
   "projectOwner": "JoshuaKGoldberg",
   "repoHost": "https://github.com",
-  "repoType": "github"
+  "repoType": "github",
+  "commitType": "docs"
```

Comment:
🤔 odd. I don't remember ever knowing that this was auto-added. Maybe relevant: https://github.com/JoshuaKGoldberg/create-typescript-app/issues/1090